server/sonar-server/src/main/java/org/sonar/server/component/ws/SearchAction.java

```diff
@@ -72,23 +75,19 @@ public SearchAction(DbClient dbClient, ResourceTypes resourceTypes, I18n i18n, U
   @Override
   public void define(WebService.NewController context) {
     WebService.NewAction action = context.createAction(ACTION_SEARCH)
-      .setSince("5.2")
-      .setInternal(true)
+      .setSince("6.3")
```

Comment:
Because if we keep the same version as before, users will think that they can use it in 5.2, but it's not true as the contract has changed.
It's a personal choice but I think if something has changed when setting a internal WS to public we should bump the version